### ISSUE TITLE
Is the project up-to-date?

### ISSUE THREAD

**Issue by AFcgi** [Keemotion/GPAC4iOS#2]
I couldn't build this lib.

**Comment by smadbe** [comment]
For info, we are still building and running an app in production with this library, so it should work somehow with the right flag and usage.

**Comment by AFcgi** [comment]
Which flags do you use? Do you build the library with Xcode 7 and iOS 9?

**Comment by smadbe** [comment]
We are compiling TS2MP4 which using this one. Flags should be enforced by pods.
Using XCode 7 and iOS8.

**Comment by AFcgi** [comment]
I fix some linked alias in your TS2MP4 demo project. But I can't compile a lib for device and simulator. It is possible that you upload a compiled ts2mp4-library (include device and simulator architecture) to the git?

**Comment by smadbe** [comment]
No, we are not gonna add binaries to the git. If you use it with cocoapods, you don't need to use it as an external lib and we do not do it.
